CONTRIBUTING.md

```diff
@@ -20,10 +20,36 @@ Repository is splitted into two packages:
 
 ## Testing your changes
 
+### Directly
+
 You can test your changes by calling `cli.js` directly from the cloned repository. You need to make sure the version of React Native matches the one present in devDependencies of the CLI. Otherwise, you may get unexpected errors.
 
 ```sh
-node /path/to/cloned/project/packages/cli/build/index.js
+node /path/to/cloned/project/packages/cli/build/index.js <command>
+```
+
+For example:
+```sh
+# Instead of react-native run-android
+node /path/to/cloned/project/packages/cli/build/index.js run-android
+```
+
+### Using yarn link
+
+Thanks to the modular nature of the cli, when you modify it, you only need to link the changed "sub-package". For example, if I change the `run-android` command, all I need to do to test it in a fresh project is:
+
+```sh
+cd /path/to/cloned/project/packages/platform-android
+yarn link
+# Should output something like:
+yarn link v1.19.0
+success Registered "@react-native-community/cli-platform-android".
+info You can now run `yarn link "@react-native-community/cli-platform-android"` in the projects where you want to use this package and it will be used instead.
+
+cd /my/new/react-native/project/
+yarn link "@react-native-community/cli-platform-android"
+
+npx react-native run-android
 ```
```

Comment:
You can see this gets pretty complicated and only affects a single package. We need an npm-script that would `cd` to every package and run `yarn link` there, so that folks can then run:
```
yarn link @react-native-community/cli-tools @react-native-community/cli-platform-ios  @react-native-community/cli-platform-android @react-native-community/cli @react-native-community/cli-types
```

Then we could write something like:

### Using yarn link

CLI is a monorepo with multiple packages interacting with each-other. In order to effortlessly test your changes you'll need to "link" them using `yarn link` for every package. You can do so with this command:
```
yarn link-packages
```
And then, in your test project, run:
```
yarn link @react-native-community/cli-tools @react-native-community/cli-platform-ios  @react-native-community/cli-platform-android @react-native-community/cli @react-native-community/cli-types
```
From now on, your test project will transparently use the CLI from symlinked packages.

Reply:
Yeah, I remember running into it when I did `link/cli` and ended up not seeing my Android changes.

Reply:
Ok, I'll work on that !